### PR TITLE
[YUNIKORN-938] Admission controller: remove v1beta1 usage

### PR DESCRIPTION
**Pull request by kingamarton**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - http://yunikorn.apache.org/community/how_to_contribute   

### What type of PR is it?
* [ ] - Bug Fix
* [X] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-938

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: pkg/plugin/predicates/predicate_manager.go [hunk 1]
```diff
@@ -23,9 +23,9 @@ import (
 	"fmt"
 
 	"go.uber.org/zap"
+	admissionV1 "k8s.io/api/admission/v1"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/kube-scheduler/config/v1beta1"
```
Comment (craigcondit) [inline pkg/plugin/predicates/predicate_manager.go:28]: I think this should be "k8s.io/kube-scheduler/config"...

File: scripts/generate-groups.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ Usage: $(basename "$0") <generators> <output-package> <apis-package> <groups-ver
 
 
 Examples:
-  $(basename "$0") all             github.com/example/project/pkg/client github.com/example/project/pkg/apis "foo:v1 bar:v1alpha1,v1beta1"
```
Comment (craigcondit) [inline scripts/generate-groups.sh:48]: These changes shouldn't be in this PR, as they aren't relevant.

**Review by craigcondit — CHANGES_REQUESTED** [review]

File: pkg/plugin/predicates/predicate_manager.go (hunk 1 above)
Comment (wilfred-s) [reply pkg/plugin/predicates/predicate_manager.go:28]: This is not related to the admission controller either. This is linked to the plugins. probably should not be part of this change either.

**Review by craigcondit — APPROVED** [review]
LGTM.

**Comment by craigcondit** [comment]
Closed via 799fc5a882f9059069080e6345c5309017fe9703.

**Pull request closed without merging**